Lux/lux_files/spiral_audio.json

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
Does this scene work? It doesn't appear in scenes.json

Reply:
Yeah I removed it.